index.ts

```diff
@@ -46,7 +47,7 @@ ${recommendations.reduce(
       <img src="${imageUri}" class="${classes.image}" alt="${title}" />
     </div>
     <div>
-      <h2 class="${classes.category}">${category}</h2>
+      ${hideCategory ? '' : `<h2 class="${classes.category}">${category}</h2>`}
```

Comment:
Do you think they want this configurable on the page?

Reply:
Sounds like they want it off until they fix the quality of their data. I think either you should enable it once the data is better or maybe tie it to some global AEM config that an author could enable.